### PR TITLE
CONSOLE-3081: swapped labels out for details view and modal

### PR DESCRIPTION
**Pull request by mattnolting**
Current light

labels-light-current

Current dark

labels-dark-current

Updated light

labels-light-update

Updated dark

labels-dark-update

This PR also contains a layout shift for details tab

Original

original

Update

update

### CODE REVIEW

**Comment by jcaianirh** [comment]
QE Approver:
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

**Comment by yapei** [comment]
@yanpzhan will verify

**Comment by yanpzhan** [comment]
@mattnolting Hi Matt, on the cluster launched by cluster-bot, I found the "No labels" lays in 2 lines, is it caused by the code change in this pr?
Screenshot from 2022-04-27 17-13-01
Screenshot from 2022-04-27 17-12-22

File: frontend/public/components/monitoring/labels.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
+import { Label as PfLabel } from '@patternfly/react-core';
 
 export const Label = ({ k, v }) => (
-  <div className="co-m-label co-m-label--expand" key={k}>
```
[the remark this answers is not in the document]
Comment (mattnolting) [reply frontend/public/components/monitoring/labels.tsx:6]: Thank you!

File: frontend/public/components/utils/_label.scss [hunk 2]
```diff
@@ -25,6 +25,33 @@
...
 
+// override label and label group default max widths
+.pf-c-label-group,
+.pf-c-label-group__main,
+.pf-c-label-group__list,
+.pf-c-label-group__list-item,
+.pf-c-label {
+  max-width: 100%;
+}
+
```
Comment (sg00dwin) [inline frontend/public/components/utils/_label.scss:36]: If this is needed we should scope this to a custom style. We are moving away from overriding PatternFly rules directly.

File: frontend/public/components/utils/_label.scss [hunk 3]
```diff
@@ -25,6 +25,33 @@
...
+.pf-c-label {
+  .pf-c-label__text {
+    --pf-c-label__text--MaxWidth: 100%;
+  }
+}
```
Comment (sg00dwin) [inline frontend/public/components/utils/_label.scss:41]: Same comment as above

File: frontend/public/components/utils/_label.scss [hunk 4]
```diff
@@ -44,11 +69,19 @@ tags-input .host:active {
 }
 
 tags-input .tags {
+  align-items: flex-start;
```
Comment (sg00dwin) [inline frontend/public/components/utils/_label.scss:72]:
```suggestion
  align-content: flex-start;
```
`align-items` can be replaced with `align-content` which will prevent the extra vertical spacing.
Screen Shot 2022-04-27 at 2 24 53 PM
Screen Shot 2022-04-27 at 2 25 13 PM

**Comment by sg00dwin** [comment]
@mattnolting I found an issue with the label tooltip generated on list pages. It includes the `<a href` markup so its styled as a link.    
Screen Shot 2022-05-10 at 3 52 21 PM (2)

**Comment by jcaianirh** [comment]
@mattnolting can you squash your commits so we can merge this please

**Comment by mattnolting** [comment]
@jcaianirh squashed!

**Review by jhadvig — APPROVED** [review]
@mattnolting change looks good 👍 
/approve

**Pull request merged**